<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radar Terminal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="num">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Num</string>
    </property>
   </widget>
   <widget class="QLabel" name="distlong">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>9</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Longitudinal (x) coordinate&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>DistLong</string>
    </property>
   </widget>
   <widget class="QLabel" name="distlat">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>9</y>
      <width>33</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lateral (y) coordinate&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>DistLat</string>
    </property>
   </widget>
   <widget class="QLabel" name="vrellong">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>9</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Relative velocity in longitudinal direction (x)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>VrelLong</string>
    </property>
   </widget>
   <widget class="QLabel" name="vrellat">
    <property name="geometry">
     <rect>
      <x>477</x>
      <y>9</y>
      <width>33</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Relative velocity in lateral direction (y)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>VrelLat</string>
    </property>
   </widget>
   <widget class="QLabel" name="rcs">
    <property name="geometry">
     <rect>
      <x>594</x>
      <y>9</y>
      <width>23</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Radar cross section&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string> RCS</string>
    </property>
   </widget>
   <widget class="QLabel" name="angel">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>9</y>
      <width>27</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dynamic property indicate if it is moving or&lt;/p&gt;&lt;p&gt;not&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Angel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lenght">
    <property name="geometry">
     <rect>
      <x>829</x>
      <y>9</y>
      <width>33</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Length</string>
    </property>
   </widget>
   <widget class="QLabel" name="width">
    <property name="geometry">
     <rect>
      <x>946</x>
      <y>9</y>
      <width>28</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="class_2">
    <property name="geometry">
     <rect>
      <x>1063</x>
      <y>9</y>
      <width>25</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Class</string>
    </property>
   </widget>
   <widget class="QLabel" name="propertis">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>9</y>
      <width>43</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Propertis</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionCAN_Connect"/>
    <addaction name="actionDisconnect_CAN"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionSet_radar_config"/>
    <addaction name="actionGet_radar_config"/>
    <addaction name="actionSet_radar_filter"/>
    <addaction name="actionPosition_view"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
   <addaction name="actionCAN_Connect"/>
   <addaction name="actionDisconnect_CAN"/>
   <addaction name="actionGet_radar_config"/>
   <addaction name="actionSet_radar_config"/>
   <addaction name="actionSet_radar_filter"/>
   <addaction name="separator"/>
   <addaction name="actionPosition_view"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
   <property name="toolTip">
    <string>Clear data</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCAN_Connect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/CANicon_on.png</normaloff>:/images/CANicon_on.png</iconset>
   </property>
   <property name="text">
    <string>CAN_Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDisconnect_CAN">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/CANicon_off.png</normaloff>:/images/CANicon_off.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect_CAN</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionGet_radar_config">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/set_radar_config.png</normaloff>:/images/set_radar_config.png</iconset>
   </property>
   <property name="text">
    <string>Get_radar_config</string>
   </property>
  </action>
  <action name="actionSet_radar_config">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/get_radar_config.png</normaloff>:/images/get_radar_config.png</iconset>
   </property>
   <property name="text">
    <string>set_radar_config</string>
   </property>
   <property name="toolTip">
    <string>Set_radar_config</string>
   </property>
  </action>
  <action name="actionSet_radar_filter">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/set_radar_filter.png</normaloff>:/images/set_radar_filter.png</iconset>
   </property>
   <property name="text">
    <string>set_radar_filter</string>
   </property>
  </action>
  <action name="actionPosition_view">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/position view.png</normaloff>:/images/position view.png</iconset>
   </property>
   <property name="text">
    <string>position view</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="terminal.qrc"/>
 </resources>
 <connections/>
</ui>
